src/calendar/tests/unit/Calendar.spec.tsx

```diff
@@ -1,6 +1,8 @@
 const { registerSuite } = intern.getInterface('object');
 const { assert } = intern.getPlugin('chai');
 
+import mockedEnv from 'mocked-env';
```

Comment:
this does seem to be a bit on the heavy side for simply changing a property on process.env?

Reply:
The package is only 100 lines :) But I see your point.

Reply:
it's not just 100 lines as it's not dependency free itself, for example it has `ramda` as a dependency among others 🙂.